src/global.f90

```diff
@@ -3421,8 +3435,8 @@ subroutine LoadManagement(FullName)
         end if
         read(fhandle, *) TempInt  ! Canopy cover (%) after cutting
         call SetManagement_Cuttings_CCcut(TempInt)
+        ! Next line NO LONGER CONSIDERED since AquaCrop version 7.1
```

Comment:
This comment might be confusing, since the next line is removed instead of commented?

Reply:
I think that means that the line is still in the input file ( .MAN file I guess). So we should still read the line to get to the next one, but this parameter is not considered anymore, hence L3425 was removed.